TODO.md

```diff
@@ -0,0 +1,17 @@
+Decide on a component structure: 1. Have Form control state and pass props to FormSection which will build inputs as FormItems 2. Build inputs directly in Form
```

Comment:
Use eslint + learn eslint

Reply:
I'd also add another task for either adding some css or getting css from some library or something

Reply:
Whew, well I think I am using it now. I am not confident.